### PR TITLE
Görli contracts deployment

### PR DESCRIPTION
**Pull request by nkuba**
Görli became a recommended test network after Ropsten's deprecation notice ([link](https://blog.ethereum.org/2022/06/21/testnet-deprecation/)).

In this PR we update deployment scripts and configuration to support Görli.

The contracts were already deployed to Görli:
`@keep-network/tbtc-v2@0.1.1-goerli.0`

### CODE REVIEW

File: solidity/hardhat.config.ts [hunk 1]
```diff
@@ -173,17 +178,20 @@ const config: HardhatUserConfig = {
...
+      goerli: 0,
       mainnet: "0xB3726E69Da808A689F2607939a2D9E958724FC2A",
     },
     keepCommunityMultiSig: {
       default: 5,
+      goerli: 0,
       mainnet: "0x19FcB32347ff4656E4E6746b4584192D185d640d",
     },
     treasury: {
       default: 6,
```
Comment (michalinacienciala) [inline solidity/hardhat.config.ts:190]: Don't we need `goerli: 0,` here as well? When I'm running the deploy on goerli, I'm getting:
```
Error: ERROR processing /home/runner/work/tbtc-v2/tbtc-v2/solidity/deploy/05_deploy_bridge.ts:
Error: invalid address (argument="address", value=undefined, code=INVALID_ARGUMENT, version=address/5.5.0) (argument="_treasury", value=undefined, code=INVALID_ARGUMENT, version=abi/5.5.0)
```

**Review by michalinacienciala — COMMENTED** [review]
Generally, the changes look good to me. But there is some problem with the deployment on `goerli` - I'm getting:
```
initializing Bridge as ECDSA wallet owner
An unexpected error occurred:
Error: ERROR processing /home/runner/work/tbtc-v2/tbtc-v2/solidity/deploy/16_initialize_wallet_owner.ts:
TypeError: Cannot read property 'length' of undefined
```

This doesn't seem to be directly caused by the changes in this PR, it looks more like a problem with https://github.com/keep-network/keep-core/pull/3061. @nkuba, do you think that merging of https://github.com/keep-network/keep-core/pull/3096 may help?

**Comment by nkuba** [comment]
> Generally, the changes look good to me. But there is some problem with the deployment on `goerli` - I'm getting:
> 
> ```
> initializing Bridge as ECDSA wallet owner
> An unexpected error occurred:
> Error: ERROR processing /home/runner/work/tbtc-v2/tbtc-v2/solidity/deploy/16_initialize_wallet_owner.ts:
> TypeError: Cannot read property 'length' of undefined
> ```
> 
> This doesn't seem to be directly caused by the changes in this PR, it looks more like a problem with [keep-network/keep-core#3061](https://github.com/keep-network/keep-core/pull/3061). @nkuba, do you think that merging of [keep-network/keep-core#3096](https://github.com/keep-network/keep-core/pull/3096) may help?

Interesting... We need to fix it in a separate PR to keep-core.

**Comment by michalinacienciala** [comment]
> Actually [bfd629c](https://github.com/keep-network/tbtc-v2/commit/bfd629cfe48e7c781572acf1a8d1417e448cf466) should fix it.

Yep, I no longer see that error. There's a different error now though, see my comment https://github.com/keep-network/coverage-pools/pull/215#issuecomment-1205090989. Usually such errors were caused by some problems with the config - when deployer account from dependencies did not match the one used currently for deploy. But it doesn't seem like it this time... Any ideas?

**Review by michalinacienciala — APPROVED** [review]
Deployment on goerli verified in job https://github.com/keep-network/tbtc-v2/runs/7744385930?check_suite_focus=true (workflow uses code combined from #387 and #382). Works as expected

**Pull request merged**